www/includes/dbtypes.php

```diff
@@ -164,6 +164,9 @@
 );
 */
 
+// epobject types
+define ('GLOSSARY_ITEM', 2);
```

Comment:
This could go right at the top of the file, where there is a comment about `eptype` that this could replace.